cmd/swarm/main.go

```diff
@@ -181,6 +181,10 @@ var (
 		Usage:  "Number of recent chunks cached in memory (default 5000)",
 		EnvVar: SWARM_ENV_STORE_CACHE_CAPACITY,
 	}
+	SwarmResourceRawFlag = cli.BoolFlag{
+		Name:  "raw",
```

Comment:
This flag is a bit too little descriptive for my taste. In the context of swarm, there's a lot which could be raw. Maybe something which better connects it to MRU?

Reply:
I agree.

Any ideas?
maybe `--verbatim`

Reply:
or prefix with `MRU` (whatever case feels better appropriate)?

Reply:
I'll go for `--rawmru`

Reply:
fixed 464798